.chloggen/remove_time_from_container_attributes.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/stanza
```

Comment:
It's probably more meaningful for users if we set the component to `receiver/filelog`? Especially that the feature gate has `filelog` in its name.